application/views/notes/edit.php

```diff
@@ -47,7 +47,7 @@
 	</div>
 
 	<input type="hidden" name="id" value="<?php echo $id; ?>" />
-	<button type="submit" value="Submit" class="btn btn-primary">Save</button>
```

Comment:
Many save buttons are in the form "Save X", where X is a QSO or whatever. Using the same format for notes.